### DISCUSSION TITLE
Looking for (best) working integration for Ortenaukreis (Germany)

### DISCUSSION THREAD

**Discussion by schwallj** [mampfes/hacs_waste_collection_schedule#2008] [Q&A]
I'm looking for the best way to integrate https://www.abfallwirtschaft-ortenaukreis.de/abfallkalender-abfuhrtermine/abfallkalender-strauchgutabfuhrtermine into home assistant.

Currently the only way seems to download and locally store a static ICS file. Does anyone know a better way?

**Comment by 5ila5** [comment]
ortenaukreis is supported by the [Abfall.IO / AbfallPlus.de](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/abfall_io.md) source and the [app_abfallplus_de source](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/app_abfallplus_de.md) with `app_id: de.k4systems.abfallappog`

note the app_abfallplus_de source had some breaking issues and aren't fixed in the latest release (see #1989)

**Reply by schwallj** [reply]
Thanks a lot. I didn't find [app_abfallplus_de source](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/app_abfallplus_de.md) myself. I'll see if everything is in order when the update is coming through.
